packages/react/src/Pagination/Pagination.tsx

```diff
@@ -75,17 +100,22 @@ export const Pagination = forwardRef(
     return (
       <nav
         {...restProps}
-        aria-labelledby={visuallyHiddenLabelId}
+        aria-labelledby={accessibleNameId || visuallyHiddenLabelId || 'ams-pagination-a11y-label'}
         className={clsx('ams-pagination', className)}
         ref={ref}
       >
-        <span className="ams-visually-hidden" id={visuallyHiddenLabelId}>
-          {visuallyHiddenLabel}
+        <span
+          className="ams-visually-hidden"
+          id={accessibleNameId || visuallyHiddenLabelId || 'ams-pagination-a11y-label'}
```

Comment:
Why change where we define default values?
(Same for `previousAccessibleName` and `nextAccessibleName`.)

Reply:
Never mind I should have read the description first.